### DISCUSSION TITLE
Button size를 추가했어요!

### DISCUSSION THREAD

**Discussion by sumi-0011** [probrainteam/GoaTani-client#6] [General]
![image](https://user-images.githubusercontent.com/49177223/229806483-64e0a70e-9932-403b-af5a-34a6dbeffa85.png)

width는 full로, 
height는 
```
lg : 60px
md : 50px
sm : 40px
xs : 30px 
```

**Comment by jeonbyeongmin** [comment]
네엡 좋습니다~
